### ISSUE TITLE
Backend API: create 'authentication' router

### ISSUE THREAD

**Issue by spencer-carver** [aymanjebril2/finanseer#13]
reference https://github.com/aymanjebril2/finanseer/wiki/API-Contract#authentication for what needs to be implemented

**Comment by spencer-carver** [comment]
NOTE: this router should accept and validate the parameters from the client, and echo a success if everything passes.

There will be a follow-up issue after #15 is completed.
